drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -506,14 +506,13 @@ def assign_unit(unit:, start_date:, user:, active_referral: nil)
 
     # include project id here since it may not be available during after_save hooks due to WIP
     self.unit_occupancy_changes = { project_id: unit.project_id, unit_type: unit.unit_type, user_id: user.id } if unit.unit_type
-    unit_occupancies.build(
-      unit: unit,
-      occupancy_period_attributes: {
-        start_date: start_date,
-        end_date: nil,
-        user: user,
-      },
+    occupancy = unit_occupancies.build(unit: unit)
```

Comment:
This change is necessitated by the change I'm proposing in `unit_occupancy.rb` -- see that thread for discussion/other alternatives.